index.html

```diff
@@ -0,0 +1,214 @@
+                <li>html</li>
+            </ul>
+            <button type="button" class="btn-popup">
+                See Project
+            </button>
+        </article>
+        <article class="card-work">
+          <div class="card-image">
+              
+          </div>
+         <h2 class="card-title">
+                  Multi-Post Stories 
+              </h2>
+              <h2>
+                  Gain+Glory
+              </h2>
+          <ul class="card-tags">
+              <li>React</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+          </ul>
+          <button type="button" class="btn-popup">
+              See Project
+          </button>
+      </article>
+      <article class="card-work">
+        <div class="card-image">
+            
+        </div>
+        <h2 class="card-title">
+                  Multi-Post Stories 
+              </h2>
+              <h2>
+                  Gain+Glory
+              </h2>
+        <ul class="card-tags">
+            <li>React</li>
+            <li>css</li>
+            <li>JavaScript</li>
+            <li>html</li>
+        </ul>
+        <button type="button" class="btn-popup">
+            See Project
+        </button>
+    </article>
+
+    <article class="card-work">
+      <div class="card-image">
+          
+      </div>
+      <h2 class="card-title">
+                  Multi-Post Stories 
+              </h2>
+              <h2>
+                  Gain+Glory
+              </h2>
+      <ul class="card-tags">
+          <li>React</li>
+          <li>css</li>
+          <li>JavaScript</li>
+          <li>html</li>
+      </ul>
+      <button type="button" class="btn-popup">
+          See Project
+      </button>
+  </article>
+
+  <article class="card-work">
+    <div class="card-image">
+        
+    </div>
+   <h2 class="card-title">
+                  Multi-Post Stories 
+              </h2>
+              <h2>
+                  Gain+Glory
+              </h2>
+    <ul class="card-tags">
+        <li>React</li>
+        <li>css</li>
+        <li>JavaScript</li>
+        <li>html</li>
+    </ul>
+    <button type="button" class="btn-popup">
+        See Project
+    </button>
+</article>
+      </div>
+  </section>
+      <section class="container-flex about-myself" id="about-myself-id">
+        <div class="container">
+            <h1>About me</h1>
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+            <button type="button2">
+                Get My Resume
+            </button>
+        </div>
+        <ul class="categorie-about-myself">
+            <li>
+                <article class="card-categories">
+                    <div class="categorie-image">
```

Comment:
- [ ] Kindly position your background images properly so that they don't appear to be colliding  with other elements in your `about me` section

![image](https://user-images.githubusercontent.com/98453979/212100412-14402d11-a745-4804-9983-1470788b5f22.png)

### Checkout Figma Design

![image](https://user-images.githubusercontent.com/98453979/212100558-5f418750-72b1-41c0-88d0-1d0defe479f3.png)